### ISSUE TITLE
Undefined index?

### ISSUE THREAD

**Issue by Flex-WKD** [bootstrapbay/contact-form#1]
Hello,

I just copied your index.php file to my index file and not changed a single line (yet) and I'm getting this error? I haven't changed any of the file/code.

Notice: Undefined index: submit in /public_html//contactus/index.php on line 2

**Comment by imti321** [comment]
I also copied the code and got this error.I was getting this error when i copied this code to my index file then i changed index and copied only the form code nothing else and i get same error
![contact form](https://cloud.githubusercontent.com/assets/4270726/5991456/a5578f68-aa16-11e4-99c6-2379a30b3113.jpg)

**Comment by cyda89** [comment]
Have you guys set the "$to" to your desired mail adress?
Set it to youradress@yourprovider.com/net/whatever.
For example: 
$to = 'YOURADRESS@gmail.com'; 

regards

**Comment by Flex-WKD** [comment]
@cyda89 yep already edited that still no luck :(

**Comment by cyda89** [comment]
@Flex-WKD 
do you have php installed on your webserver? is phpmail working?
pushing the index.php to your webserver without copying anything and only changing the $to. if still doesnt work, you probably need to configure your webserver.

**Comment by Flex-WKD** [comment]
@cyda89 Yeah it's enabled and installed and all other scrips and mail scripts are working fine my host said it's the code.

 I'll try on another server with my other provider and see how that goes.

**Comment by yateendra** [comment]
How to add ajax into it?

**Comment by morettimatheus** [comment]
The problem is this code is wrong. You should never echo variables without testing their existance.
To fix it, simply add an if(isset($var)) {echo $var}
where var is obviously your desired variable

**Comment by mdogan1** [comment]
Hello I just copied  code on my index and  when i preview in the browser I got something like on the image.

<img width="1065" alt="ekran resmi 2015-10-12 oo 10 41 28" src="https://cloud.githubusercontent.com/assets/6609709/10424272/1d82d0e6-70ce-11e5-8bbe-a7bc9cd20274.png">

**Comment by MaliciaM** [comment]
Hi.

I'm a to-the-core-noob in php... but I think the "Undefined variable: errName " and so on are there because they weren't defined at the beginning. I had the same problem and after some time, I simply added at the very beginning of the php-part.

$errName ='';
$errEmail ='';
$errMessage ='';
$errHuman ='';
$result ='';

At least it looks like it works now for me. If this is the wrong solution could you tell me what is the correct solution? Thanks :)

**Comment by vengatowen** [comment]
@mustyprogrammer   You need to rename the html file in to .php so that it will work.

**Comment by ErikThiart** [comment]
Code does not work.

**Comment by panneerselvam03** [comment]
Code is not working for me...

**Comment by inaztulip** [comment]
Code does not work for me~

**Comment by fabeka** [comment]
code not, not not

**Comment by fabeka** [comment]
please help. You are doing a good job
